src/rgw/rgw_sal_store.h

```diff
@@ -233,6 +234,8 @@ class StoreObject : public Object {
     virtual const std::string &get_instance() const override { return state.obj.key.instance; }
     virtual bool have_instance(void) override { return state.obj.key.have_instance(); }
     virtual void clear_instance() override { state.obj.key.instance.clear(); }
+    virtual jspan_context& get_trace() { return trace_ctx; }
+    virtual void set_trace (jspan_context&& _trace_ctx) { trace_ctx = _trace_ctx; }
```

Comment:
why do you do "copy" here and "move" in: `src/rgw/rgw_sal_filter.h`

Reply:
This is a move call (&&), so it shouldn't need std::move.  I don't know about the filter version, though, because it's calling through.

Reply:
the filter also accept `jspan_context&&` as a parameter